### PR TITLE
Link style in text

### PR DESCRIPTION
**Pull request by WilcoFiers**

### CODE REVIEW

File: _drafts/SC1-4-1-link-style-in-text.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ This rule checks that links that are embedded in a block of text can be distingu
...
 
 ## Assumptions
 
-- This test assumes that the 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of G183 technique, but is not specified in the 1.4.1 success criterion.
-- This tests assumes that one of the following shorthand CSS-properties (or the related expanded properties) is used to make the link visually evident: `background`, `border`, `color`, `font`, or `text-decoration`.
-- This test assumes that any use of border will make links sufficiently distinguishable
-- This test assumes that the different font is presented to the user.
-- Use of a `border`, of 1 or more pixels, not set to none, and not with a color of transparent, is assumed to be distinguishable
+- This test assumes that the 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of technique G183, but is not specified in the 1.4.1 success criterion.
+- This tests assumes that one of the following shorthand CSS properties (or the related expanded properties e.g. `background-color`) is used to make the link visually evident: `background`, `color`, `font`, or `text-decoration`.
```
Comment (EmmaJP) [inline _drafts/SC1-4-1-link-style-in-text.md]: "This test"

File: _drafts/SC1-4-1-link-style-in-text.md [hunk 2]
```diff
@@ -77,11 +75,11 @@ Else continue with [Step 2](#step-2-link-contrast)
 
 Test mode: [automatic][AUTO]
 
-Determine `color` and `background-color` of the link and it's block-like ancestor. If the background-color is transparent, use getElementsFromPoint to locate the closest underlying element that does have a `background-color` and use that value.
+Determine `color` and `background-color` of the link and it's block-like ancestor. If the background-color is transparent, locate the closest underlying element that does have a `background-color` and use that value. (GetElementsFromPoint can be used for this.)
```
Comment (EmmaJP) [inline _drafts/SC1-4-1-link-style-in-text.md]: This is useful to know for 1.4.3 - thanks

File: _drafts/SC1-4-1-link-style-in-text.md (hunk 1 above)
Comment (cpandhi) [inline _drafts/SC1-4-1-link-style-in-text.md]: May be want to say >> This test assumes that the 3:1 contrast difference between the link text and the surrounding text << to make more clear.

File: _drafts/SC1-4-1-link-style-in-text.md (hunk 2 above)
Comment (cpandhi) [inline _drafts/SC1-4-1-link-style-in-text.md]: Looks good

**Pull request merged**